properties.schema

```diff
@@ -66,6 +66,15 @@
       "help": "This is the instruction text",
       "translatable": true
     },
+    "ariaQuestion": {
+      "type": "string",
+      "required": false,
+      "default": "",
+      "inputType": "Text",
+      "validators": [],
+      "help": "This is the ariaQuestion text",
```

Comment:
I'm not sure an AAT user will understand what ariaQuestion text is. Suggestion:

This will be read out by screen readers instead of reading the title, body & instruction fields when focusing on the options. To be clear and concise, ensure the text encompasses only the question associated.

Reply:
Good point, the help text has now been updated.